File: app/src/main/java/develop/y/zhzl/list/view/IListView.java
package develop.y.zhzl.list.view;

import java.util.List;

import develop.y.zhzl.list.model.ListModel;
import framework.mvp.BaseView;

/**
 * by y on 2016/8/7.
 */
public interface IListView extends BaseView<List<ListModel>> {

    void hideEmptyView();

    void isShowEmptyView();

    void removeAllAdapter();
}

File: app/src/main/java/develop/y/zhzl/list/view/IListView.kt
package develop.y.zhzl.list.view

import develop.y.zhzl.list.model.ListModel
import framework.mvp.BaseView

/**
 * by y on 2016/8/7.
 */
interface IListView : BaseView<List<ListModel>> {

    fun hideEmptyView()

    fun isShowEmptyView()

    fun removeAllAdapter()
}
